XBVC/emitters/templates/c_interface.jinja2

```diff
@@ -219,6 +219,15 @@ static int xbvc_decode_message(uint8_t *ib,
     return 0;
 }
 
+static void _default_handler(void *msg)
+{
+    XBVCTOUCH(msg);
+}
+
+{% for msg in dec_msgs %}
+void xbvc_handle_{{msg.name}}(struct x_{{msg.name}} *msg) __attribute__((weak, alias("_default_handler")));
```

Comment:
While trying to read some stuff to explain what's going on here, I see that the example of the alias attribute  given in some [gcc documentation](https://gcc.gnu.org/onlinedocs/gcc-4.7.2/gcc/Function-Attributes.html) uses a double underscore in the aliased function name. Is there any special significance between the double- and single-underscores in such contexts?

Reply:
Mostly historical precedent.